src/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -396,6 +398,7 @@ public abstract partial class HttpHeaders : System.Collections.Generic.IEnumerab
         public void Add(string name, string value) { }
         public void Clear() { }
         public bool Contains(string name) { throw null; }
+        protected virtual void EnsureHeaders() { }
```

Comment:
Do we need this to be protected?  Could it instead be either "internal" or "private protected"?  In which case it wouldn't be public API and wouldn't need to be in the ref.